tests/unit/replication/ReplicateObject.spec.js

```diff
@@ -1,166 +0,0 @@
-const assert = require('assert');
```

Comment:
we should port these tests to some `tests/unit/replication/UpdateReplicationStatus.js` ?